cdap-proto/src/main/java/co/cask/cdap/proto/NamespaceConfig.java

```diff
@@ -121,4 +121,35 @@ public String toString() {
       ", keytabURI='" + keytabURI + '\'' +
       '}';
   }
+
+  /**
+   * Pretty print only non-empty field of {@link NamespaceConfig}
+   *
+   * @return the String to print
+   */
+  public String prettyPrintCLI() {
```

Comment:
If this is only used in CLI, let's move it to the CLI classes. classes in cdap-proto are public, so we can't make incompatible changes here later. Should not add CLI specific code here